import-dashboards.py

```diff
@@ -92,6 +108,33 @@ def delete_api_key(upgrade):
     con.close()
 
 
+def fix_cloudwatch_datasource():
+    """
+    Replaces incorrect CloudWatch datasource stored as JSON string with correct JSON object.
+    """
+
+    con = sqlite3.connect(GRAFANA_DB_DIR + '/grafana.db')
+    cur = con.cursor()
+
+    found = False
+    cur.execute("SELECT id, json_data FROM data_source WHERE name = 'CloudWatch'")
```

Comment:
likely, it is impossible to create two datasources with the same name

Reply:
But it may be removed by a user.